test/profile.update.test.ts

```diff
@@ -13,7 +13,7 @@ let token: string = '';
 describe('update-profile', () => {
   before(done => {
     const currentuser = {
-      email: 'test4@example.com',
+      email: 'test2@example.com',
       password: '1111@aa',
```

Comment:
again why ??

Reply:
this is done before the user they passed here had no roleId so i changed the user to track the issue but locally it is good